apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktKopierService.kt

```diff
@@ -0,0 +1,48 @@
+package no.nav.etterlatte.behandling.aktivitetsplikt
+
+import no.nav.etterlatte.behandling.aktivitetsplikt.vurdering.AktivitetspliktAktivitetsgrad
+import no.nav.etterlatte.behandling.aktivitetsplikt.vurdering.AktivitetspliktAktivitetsgradDao
+import no.nav.etterlatte.behandling.aktivitetsplikt.vurdering.AktivitetspliktUnntak
+import no.nav.etterlatte.behandling.aktivitetsplikt.vurdering.AktivitetspliktUnntakDao
+import java.util.UUID
+
+class AktivitetspliktKopierService(
+    private val aktivitetspliktAktivitetsgradDao: AktivitetspliktAktivitetsgradDao,
+    private val aktivitetspliktUnntakDao: AktivitetspliktUnntakDao,
+) {
+    private fun hentVurderingForBehandling(behandlingId: UUID): AktivitetspliktVurdering? {
+        val aktivitetsgrad = aktivitetspliktAktivitetsgradDao.hentAktivitetsgradForBehandling(behandlingId)
+        val unntak = aktivitetspliktUnntakDao.hentUnntakForBehandling(behandlingId)
+
+        if (aktivitetsgrad == null && unntak == null) {
+            return null
+        }
+
+        return AktivitetspliktVurdering(aktivitetsgrad, unntak)
+    }
+
+    fun kopierVurdering(
+        sakId: Long,
+        behandlingId: UUID,
+    ) {
+        val vurdering = hentVurderingForBehandling(behandlingId)
+        if (vurdering?.unntak != null || vurdering?.aktivitet != null) {
+            return
+        }
+
+        val aktivitetsgrad = aktivitetspliktAktivitetsgradDao.hentNyesteAktivitetsgrad(sakId)
+        val unntak = aktivitetspliktUnntakDao.hentNyesteUnntak(sakId)
+        val nyesteVurdering = listOfNotNull(aktivitetsgrad, unntak).sortedBy { it.opprettet.endretDatoOrNull() }.lastOrNull()
+
+        if (nyesteVurdering != null) {
+            when (nyesteVurdering) {
+                is AktivitetspliktAktivitetsgrad ->
+                    aktivitetspliktAktivitetsgradDao.kopierAktivitetsgrad(
+                        nyesteVurdering.id,
+                        behandlingId,
+                    )
+                is AktivitetspliktUnntak -> aktivitetspliktUnntakDao.kopierUnntak(nyesteVurdering.id, behandlingId)
+            }
+        }
```

Comment:
Det er noe som skurrer litt her - vi vil i framtiden få flere perioder der vi kan få unntak til en dato, og så en vurdering når unntaket løper ut. Men det er kanskje mer mat for periodisering av vurdering.

Reply:
Jeg vil tro det er mat for periodisering av vurdering, for slik det er nå så har vi lite å gå på annet enn å hente ut den nyeste fra begge parter